cdap-ui/app/cdap/components/FastAction/SetPreferenceAction/SetPreferenceModal.js

```diff
@@ -241,10 +271,27 @@ export default class SetPreferenceModal extends Component {
         description = T.translate('features.FastAction.setPreferencesDescriptionLabel.program');
       }
     }
+
     return (
       <div>
+        {
+          this.isTitleOverflowing() ?
+            (
+              <Tooltip
```

Comment:
I think we should have the placement at the bottom. Right now it is missing the arrow and it is too far away from the title.
<img width="980" alt="screen shot 2017-01-30 at 11 51 02 am" src="https://cloud.githubusercontent.com/assets/1452845/22438906/850253e6-e6e2-11e6-8b5b-d2745b71a537.png">